### PR TITLE
LU : Few changes

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Ditch port 1935 to HTTPS (Not sure about that one, it seems that there's a cert issue, to test)

We do not need definst.

EDIT : Got effed up by the certificate, but VLC still plays.
vlcsnap-2021-10-06-16h24m24s118

### CODE REVIEW

File: channels/lu.m3u [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="ChamberTV.lu" tvg-country="LU" tvg-language="Luxembourgish" tvg-logo="https://i.imgur.com/YldMnap.jpg" group-title="",Chamber TV (1080p)
-https://media02.webtvlive.eu/chd-edge/_definst_/smil:chamber_tv_hd.smil/playlist.m3u8
+https://media02.webtvlive.eu/chd-edge/smil:chamber_tv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="eldoTV.lu" tvg-country="LU" tvg-language="Luxembourgish" tvg-logo="https://i.imgur.com/4ntor8S.png" group-title="",eldo.TV (720p)
 https://eldo-streaming.eldo.lu/eldotv/smil:eldotv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="JasminTV.lu" tvg-country="LU" tvg-language="English" tvg-logo="https://www.parsatv.com/index_files/channels/jasmintv.png" group-title="XXX",Jasmin TV (720p)
-http://109.71.162.112:1935/live/hd.jasminchannel.stream/playlist.m3u8
+https://109.71.162.112/live/hd.jasminchannel.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/lu.m3u:7]: Naughty little channel.

Works using ffplay, could not connect using Firefox.

File: channels/lu.m3u [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 #EXTINF:-1 tvg-id="JasminTV.lu" tvg-country="LU" tvg-language="English" tvg-logo="https://www.parsatv.com/index_files/channels/jasmintv.png" group-title="XXX",Jasmin TV (720p)
-http://109.71.162.112:1935/live/sd.jasminchannel.stream/playlist.m3u8
+https://109.71.162.112/live/sd.jasminchannel.stream/playlist.m3u8
```
Comment (jkemp814) [inline channels/lu.m3u:9]: Does play but VLC throwing certificate error:

[00007fc56429d3a0] gnutls tls client error: Certificate verification failure: The certificate is NOT trusted. The name in the certificate does not match the expected.

File: channels/lu.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/lu.m3u:7]: That's what I was expecting, I don't know if it plays on http://

**Comment by LaneSh4d0w** [comment]
image
image


It seems that both 'SD' and 'HD' Jasmin TV channels are on 720p. Is it useful to keep both versions ?

**Review by jkemp814 — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**